src/main/resources/reference.conf

```diff
@@ -43,6 +43,13 @@ videobridge {
     # will be used by Jicofo in the selection of a bridge for
     # a client by comparing it to the client's region.
     #region="us-west-1"
+    # The address on which the Octo relay should bind
+    #bind-address=198.51.100.1
+    # The port to which the Octo relay should bind
+    #bind-port=4096
+    # The address which controls the public address which
+    # will be part of the Octo relayId
+    #public-address=198.51.100.1
```

Comment:
If we set a default of `${videobridge.octo.bind-address}`, is it going to work as expected when `bind-address` is overriden in `application.conf`, but `public-address` isn't?

Reply:
The properties are completely independent, so they'd be overridden separately.  What would be expected?